app/views/targets/new.html.haml

```diff
@@ -0,0 +1,85 @@
+#googleMap
+%script{:src => "http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"}
+:javascript
+  function myMap() {
```

Comment:
see corrections on other file. 
also if this code is repeated move it into a Partial
http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials